releases/release-1.31/release-notes/maps/pr-122363-map.yaml

```
@@ -0,0 +1,4 @@
+pr: 122363
+releasenote:
+  text: Kubelet flags `--iptables-masquerade-bit` and `--iptables-drop-bit` were deprecated
```

```suggestion
  text: Removed Kubelet flags `--iptables-masquerade-bit` and `--iptables-drop-bit` as they were deprecated in v1.28.
```

https://github.com/kubernetes/sig-release/pull/2542#discussion_r1653158938